src/platform/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -275,6 +264,35 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
         return undefined;
     }
 
+    // Use our jupyter session to delete all the cells
+    private async deleteDumpCells() {
+        const fileValues = [...this.cellToFile.values()];
```

Comment:
cellToFile is a Map so values / keys will be an IterableIterator. Just converted to an array with spread syntax so that I could use .map on it. I think Array.from could also be used, but there isn't a perf difference that I'm aware of.